### ISSUE TITLE
The new 'Terminals' API

### ISSUE THREAD

**Issue by Jeffrey-Zutt** [mollie/mollie-api-node#328]
Hello!

First off all, thanks for maintaining this great library :-).

I was wondering if there are any plans of integrating the new 'terminals' API into this library.
https://docs.mollie.com/reference/v2/terminals-api/overview

Happy to help and raise a PR on it.

**Comment by fjbender** [comment]
Thanks for reaching out! We're always happy to take contributions :)

Once the `resource` issue on the Terminal API is covered, let's take another look at this.

**Comment by Jeffrey-Zutt** [comment]
Yeah I noticed that too, the `resource` property is missing from the actual API response right? 
(Even though it's documented here: https://docs.mollie.com/reference/v2/terminals-api/get-terminal#response)

Do you happen to have any idea when this issue will be covered? (Ballpark?)

**Comment by fjbender** [comment]
I have just been informed that one of our fantastic engineers has fixed it already :) Should be available very shortly.

**Comment by Jeffrey-Zutt** [comment]
Awesome! I'll keep you posted :-)

**Comment by Jonas4Mollie** [comment]
The resource parameter is now part of the API response, so you should be able to move ahead!

**Comment by iFlyinq** [comment]
Any updates on this one?

**Comment by janpaepke** [comment]
addition is planned for 4.2.0
